src/javascripts/components/ChatInput.js

```diff
@@ -1,14 +1,30 @@
-var React = require('react');
+import React from 'react';
 
 class ChatInput extends React.Component {
+  handleSubmit = (event) => {
+    event.preventDefault();
```

Comment:
Usually, it's best to put `event.preventDefault()` at the _end_ of a method.  The reason being: if the javascript in the method fails, then we never get to `event.preventDefault()` and the default behavior is maintained.

This is a progressive enhancement best-practice (which is admittedly less applicable in React), but is good to do whenever you can.